### ISSUE TITLE
How to diagnose why an application is not registered with dotnet-suggest

### ISSUE THREAD

**Issue by TPIvan** [dotnet/command-line-api#1822]
I have an application (framework 4.7.2 VB.NET) using System.Commandline, which ecnapsulates our company business logic. I wanted to try tab competitions and so I have installed  dotnet-suggest and added the shell startup script, but tab completitions was not working. Finally I have found that file `.dotnet-suggest-registration.txt` is not updated (`.UseDefaults()` is called). When I register the application manually (with `dotnet-suggest register`) tab completion  works as expected.

To diagnose the problem I created a brand new application with following code:

**Comment by TPIvan** [comment]
wrongly written I will create a new one

**Comment by TPIvan** [comment]
Thanks to @KalleOlaviNiemitalo I realized that the problem is not in my code. 

I checked `dotnet-suggest` source code and found that there is a condition dealing with `%temp%/system-commandline-sentinel-files` after clearing that folder everything worked as expected. 

After this I discovered the issue #1588 and realized that my problem was installing `dotnet-suggest` after the running my application first.
